Makefile

```diff
@@ -182,15 +201,40 @@ clean:	## Clean up the built artifacts
 	rm -rf submariner-operator/deploy/submariner
 	rm -f submariner-operator/pkg/subctl/operator/common/embeddedyamls/yamls.go
 
-stop-clusters:	## Removes the running kind clusters
+remove-git-repos:	## Remove local copy of upstream repositories
```

Comment:
Since you're already doing "spring cleaning", consider if all of these should be called `clean-*` instead of `remove-*` since the top level is called `clean`

Reply:
I prefer "remove" than "clean" for repos, since we actually delete the repos not cleaning them.